specfile/specfile.py

```diff
@@ -50,9 +65,14 @@ class Specfile:
         autosave: Whether to automatically save any changes made.
     """
 
+    ENCODING_ARGS: EncodingArgs = {"encoding": "utf8", "errors": "surrogateescape"}
+    _file: IO
```

Comment:
Is this really necessary? Isn't the type inferred from the first assignment to `self._file`?